packages/api/src/controllers/room.ts

```diff
@@ -0,0 +1,34 @@
+import { Router } from "express";
+import { RoomServiceClient } from "livekit-server-sdk";
+import { v4 as uuid } from "uuid";
+import { authorizer } from "../middleware";
+
+const app = Router();
+
+app.post("/", authorizer({}), async (req, res) => {
+  if (req.config.livekitHost == "") {
+    res.status(400);
+    return res.send("not enabled");
+  }
+
+  const id = uuid();
+  const svc = new RoomServiceClient(
+    req.config.livekitHost,
+    req.config.livekitApiKey,
+    req.config.livekitSecret
+  );
+  const opts = {
+    name: id,
+    // timeout in seconds
+    emptyTimeout: 30 * 60,
+    maxParticipants: 2,
+  };
+  const room = await svc.createRoom(opts);
```

Comment:
Do we not need to create a stream anymore? So there will be a separate call to have the room be livestreamed, is that it?

Reply:
That's right, since Hunter mentioned that not every room will have a live stream i've kept it separate.